### ISSUE TITLE
Handle replacement of support library references in xml resource files

### ISSUE THREAD

**Issue by cabaird** [dpa99c/cordova-plugin-androidx-adapter#9]
**I'm submitting a ...**  (check one with "x"):
- [ ] bug report
- [x] feature request
- [ ] documentation issue

# Feature request

### Handle replacement of support library references in xml resource files
Currently, this plugin replaces these library references with a match for java files.  There are some instances where these same support libraries are being referenced in xml resource files.  

Ex:
```
<android.support.constraint.ConstraintLayout
  xmlns:android="http://schemas.android.com/apk/res/android"
  xmlns:app="http://schemas.android.com/apk/res-auto"
  xmlns:tools="http://schemas.android.com/tools"
  android:layout_width="match_parent"
  android:layout_height="match_parent"
  tools:context=".MainActivity">

...
```
Should be replaced with:
```
<androidx.constraintlayout.widget.ConstraintLayout
  xmlns:android="http://schemas.android.com/apk/res/android"
  xmlns:app="http://schemas.android.com/apk/res-auto"
  xmlns:tools="http://schemas.android.com/tools"
  android:layout_width="match_parent"
  android:layout_height="match_parent"
  tools:context=".MainActivity">

...
```
This should use the same mappings as the Java files, so it should only take a change to the check for matching files to include `.xml` suffixed files.

**Comment by phyr0s** [comment]
+1

**Comment by dpa99c** [comment]
Fix published in v1.1.1
